pkg/asset/asset.go

```diff
@@ -50,3 +59,36 @@ func PersistToFile(asset WritableAsset, directory string) error {
 	}
 	return nil
 }
+
+// PersistTargets saves the target assets on disk.
+// It also removes any previous generated assets.
```

Comment:
Have we abandoned the idea of having a manifests-templates target? The backup directory plan used here only works if there is a linear progression in targets. If we have a install-config target and a manifests-templates target that are independent are both precede the manifests target, then running one of install-config and manifests-templates will wipe out the other.

Reply:
@staebler Maybe we should dump both install config and templates when the target is templates?

Reply:
That works if the templates target is run after the install-config target. If the templates target is run first, that would fail. Probably it is not a common use case to run templates first and then install-config, though. Maybe both targets dump both install-config and tempate assets. Maybe the targets do something where they don't generate the assets, but if the asset files are on-disk then they will be loaded and written back out into the new directory.